pages/signup.js

```diff
@@ -1,9 +1,9 @@
-import SignUpForm from '../components/SignUpForm';
+import SignUpForm from '../containers/SignUpForm';
 import withData from '../libraries/withData';
-import DefaultCon from '../containers/Default';
+import Layout from '../containers/Layout';
```

Comment:
It should stay with the same name Because you can understand what it's ```Default``` container. ```Layout``` doesn't have same meaning.